cram/cram_io.c

```diff
@@ -939,6 +954,7 @@ int cram_uncompress_block(cram_block *b) {
         b->method = RAW;
         return 0;
     }
+    assert(b->uncomp_size > 0); // cram_read_block should ensure this
```

Comment:
`b->uncomp_size == 0` is handled in the block above.

Reply:
Good point.  I think best is probably just to make cram_block comp_size and uncomp_size uint32_t, although this will do for now as that'd probably entail many more changes to checks elsewhere.

The spec just defines these as "itf8" which is woolly at best.  I think CRAM never intended to handle negatives in the way it does and it has an unwritten rule that 0xffffffff is -1, so there is no way for the spec to state something as unsigned.  Ghastly!  (One of my aims for CRAM4 is to be explicit and replace itf8 with u7 and s7 using something like zig-zag encoding or bit-0 sign bit.)